Sources/AppStoreConnectCLI/Commands/Certificates/ListDownloadCertificatesCommand.swift

```diff
@@ -70,19 +56,19 @@ struct ListDownloadCertificates: CommonParsableCommand {
         if let downloadPath = downloadPath {
             try certificates.forEach { (certificate: Certificate) in
                 guard let content = certificate.content else {
-                    throw CommandError.invalidContent
+                    throw CertificatesError.invalidContent
```

Comment:
🤔 This error might be better named `noContent`.
```suggestion
                    throw CertificatesError.noContent
```